win32ss/gdi/gdi32/misc/heap.c

```diff
@@ -54,16 +45,21 @@ HEAP_strdupA2W ( LPWSTR* ppszW, LPCSTR lpszA )
     if ( !*ppszW )
         return STATUS_NO_MEMORY;
     Status = RtlMultiByteToUnicodeN ( *ppszW, len*sizeof(WCHAR), NULL, (PCHAR)lpszA, len );
-    (*ppszW)[len] = L'\0';
+    (*ppszW)[len] = UNICODE_NULL;
     return Status;
 }
 
-
-VOID
-HEAP_free ( LPVOID memory )
+PWSTR FASTCALL
+HEAP_strdupA2W_buf(IN PCSTR lpszA, OUT PWSTR pszBuff, IN SIZE_T cchBuff)
```

Comment:
```suggestion
HEAP_strdupA2W_buf(_In_ PCSTR lpszA, _Out_ PWSTR pszBuff, _In_ SIZE_T cchBuff)
```
Side-note: it seems `pszBuff` is actually used as input; if so, the 2nd param annotation should be `_In_` as well (If this needs to be fixed, then the line in the header needs the fix as well.)